### PR TITLE
Accessibility changes to conf includes I-W

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: N/A

Description of Changes:
* Accessibility fixes (mostly)
* Fixed heading levels
* Fixed repeated headings (therefore, repeated ids)
* Fixed most headings marked with bold rather than heading elements
* Fixed some notes for consistency
* Fixed some tabs to spaces
* Fixed some poorly worded content
* Maybe some other stuff

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: _includes/configuration/images-upload-credentials.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
-## images_upload_credentials
+## `images_upload_credentials`
 
-The **images_upload_credentials** option lets you specify if calls to the configured [`images_upload_url`](#images_upload_url) should pass along credentials (such as cookies, authorization headers, or TLS client certificates) for cross-domain uploads. When set to `true`, credentials will be sent to the upload handler, similar to the [`withCredentials` property of `XMLHttpRequest`s](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/withCredentials).
+The **images_upload_credentials** option specifies if calls to the configured [`images_upload_url`](#images_upload_url) should pass along credentials (such as cookies, authorization headers, or TLS client certificates) for cross-domain uploads. When set to `true`, credentials will be sent to the upload handler, similar to the [`withCredentials` property of `XMLHttpRequest`s](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/withCredentials).
```
Comment (JamesToohey) [inline _includes/configuration/images-upload-credentials.md:3]:
```suggestion
The **images_upload_credentials** option specifies whether calls to the configured [`images_upload_url`](#images_upload_url) should pass along credentials (such as cookies, authorization headers, or TLS client certificates) for cross-domain uploads. When set to `true`, credentials will be sent to the upload handler, similar to the [`withCredentials` property of `XMLHttpRequest`s](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/withCredentials).
```

Thoughts?

**Review by JamesToohey — APPROVED** [review]

File: _includes/configuration/images-upload-credentials.md (hunk 1 above)
Comment (tylerkelly13) [reply _includes/configuration/images-upload-credentials.md]: Accepted your suggestion (Thanks. I was too tired when I got to this one) https://github.com/tinymce/tinymce-docs/pull/1708/commits/bb8425e34457abc59a9ca725089672fc9f8c083b

**Review by JamesToohey — APPROVED** [review]

**Review by ltrouton — APPROVED** [review]

File: _includes/configuration/menubar.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ To specify the menus that should appear on {{site.productname}}'s menu bar, the
 
 {{site.differs_for_mobile}}
 
-##### Example: `menubar` string
+### Example: Using `menubar` string
```
Comment (ashfordneil) [inline _includes/configuration/menubar.md:15]: "as a" string / boolean maybe?

**Review by ashfordneil — APPROVED** [review]

File: _includes/configuration/menubar.md (hunk 2 above)
Comment (tylerkelly13) [reply _includes/configuration/menubar.md]: Fixed by https://github.com/tinymce/tinymce-docs/pull/1708/commits/a2dd27d2c3872be75df83cdf821bb7d1f86ced82

**Review by EkimChau — APPROVED** [review]

**Pull request merged**